### PR TITLE
Added logging for stop steps.

### LINKED ISSUE #23: Print how many emulators were stopped and how long did it take.
Or print available devices before and after stop, because current output is too uniformative:

```console
java -jar /tmp/swarmer.jar stop
[Wed Jun 28 20:17:17 UTC 2017]: All emulators stopped
```
And there is no `.` (dot) at the end!!1

### PR DESCRIPTION
**Pull request by yunikkk**
Closes #23

### CODE REVIEW

File: swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt [hunk 1]
```diff
@@ -96,19 +96,27 @@ fun stopAllEmulators(
...
-                completableProcess(
-                        listOf(adb, "-s", emulator.id, "emu", "kill"),
-                        args.timeoutSeconds to SECONDS
-                )
+) {
+    val startTime = System.nanoTime()
+
+    connectedEmulators()
+            .map { emulators ->
+                log("Stop running emulators: $emulators.")
```
Comment (artem-zinnatullin) [inline swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Stopping

File: swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt [hunk 2]
```diff
@@ -96,19 +96,27 @@ fun stopAllEmulators(
...
+                            .doOnCompleted { log("Stopped emulator $emulator.") }
+                }
             }
-        }
-        .flatMapCompletable(Completable::merge)
-        .doOnError { log("Error during all emulators stop, error = $it") }
-        .doOnCompleted { log("All emulators stopped") }
-        .await()
+            .flatMapCompletable(Completable::merge)
+            .doOnError { log("Error stopping emulators, error = $it.") }
```
Comment (artem-zinnatullin) [inline swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: "Error during stopping running emulators, error $it."?

Comment (yunikkk) [inline swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Ok

File: swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt (hunk 1 above)
Comment (yunikkk) [inline swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Ok

**Comment by yunikkk** [comment]
@artem-zinnatullin PTAL

File: swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt [hunk 3]
```diff
@@ -96,19 +96,27 @@ fun stopAllEmulators(
...
-        .flatMapCompletable(Completable::merge)
-        .doOnError { log("Error during all emulators stop, error = $it") }
-        .doOnCompleted { log("All emulators stopped") }
-        .await()
+            .flatMapCompletable(Completable::merge)
+            .doOnError { log("Error during stopping emulators, error = $it.") }
+            .doOnCompleted { log("All emulators stopped.") }
+            .await()
+
+    log("Swarmer: - \"My job is done here, took ${(System.nanoTime() - startTime).nanosAsSeconds()} seconds, bye bye.\"")
```
Comment (artem-zinnatullin) [inline swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Hm, I guess we'll need to extract this log to a separate function, but we could do it later

**Pull request merged**